docs(fix): wording MTA-1561 (#202)
scaleway/docs-content: tutorials/set-up-yunohost-debian/index.mdx

- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)

> - You have a Virtual Instance running on Debian 9 (Stretch)

Reviewer: ```suggestion
- You have an Instance running on Debian 9 (Stretch)
```

- You have a domain name pointed to your instance

</Message>

## Installing Yunohost

> 1. Connect to the Virtual Instance via [SSH](https://www.scaleway.com/docs/create-and-connect-to-your-server/#-Logging-into-the-Instance).

Reviewer: ```suggestion
1. Connect to the Instance via [SSH](https://www.scaleway.com/docs/create-and-connect-to-your-server/#-Logging-into-the-Instance).
```
